fabtests/ubertest/config.c

```diff
@@ -369,6 +369,7 @@ static int ft_parse_num(char *str, int len, struct key_t *key, void *buf)
 		TEST_ENUM_SET_N_RETURN(str, len, FI_PROGRESS_MANUAL, int, buf);
 		TEST_ENUM_SET_N_RETURN(str, len, FI_PROGRESS_AUTO, int, buf);
 		TEST_ENUM_SET_N_RETURN(str, len, FI_PROGRESS_UNSPEC, int, buf);
+		TEST_ENUM_SET_N_RETURN(str, len, FI_PROGRESS_CONTROL_UNIFIED, int, buf);
```

Comment:
I think this should get removed so it can get isolated to the API change. These enums in ubertest are ones that the application can set in order to test it. If there's no support in ubertest to verify it works then we probably shouldn't add it to the enums yet.

Reply:
done